docs/modules/ROOT/pages/partials/con_hermetic_verification.adoc

```diff
@@ -1,16 +1,17 @@
 .Verification
 
-. On your application, go to the *Activity > Pipeline runs* tab.
+. From the *Applications* view, go to *Activity > Pipeline runs*.
+. Optional: In your Git repository, select the *Commits* tab, then review your related pull request (PR).
```

Comment:
Sorry for not being clearer, that Commit tab you mention is in Konflux, not GH. From whatever application their repo is associated with, they could look at the Activity tab and go to `Latest commits`, instead of `Pipeline Runs` and directing them to decipher which is related to their change.

There is a way to see some limited feedback in GH, but that would require different instructions. Example here in the Checks tab - https://github.com/mattreid/partner-catalog3/pull/2/checks?check_run_id=20577504209 There you can see that the build-container task finished successfully after 11 mins.

Reply:
There is no commits tab in GH. They could review the PR directly in GH, to see if the tasks pass or not. From their repo, click Pull Requests, click their PR, click the Checks tab, click our github app name to see its Checks details which would list the tasks and their status, but doesn't provide detailed info like logs. See the GH link above for an example of what that looks like.

There is a `Latest commits` tab in Konflux, when viewing activity within an application. Which is an alternate way to view Pipeline Runs, aggregating pipeline runs based on their association to PRs, instead of displaying a flat list of every pipeline run, which is what the `Pipeline runs` tab displays.